test/ok_test.exs

```diff
@@ -44,36 +44,57 @@ defmodule OKTest do
     assert {:error, :some_reason} = OK.failure(:some_reason)
   end
 
-  test "macro passes success value to function" do
-    report_func = fn (arg) -> send(self, arg) end
+  # test "macro passes success value to function" do
```

Comment:
OK that is what I figured. The reason I ask is just to point out the commented out cruft to be removed before merging the PR.